partiql-plan/src/main/kotlin/org/partiql/plan/v1/operator/rel/RelScan.kt

```diff
@@ -13,7 +12,7 @@ public interface RelScan : Rel {
 
     override fun getChildren(): Collection<Rel> = emptyList()
 
-    override fun isOrdered(): Boolean = getInput().getType().kind == PType.Kind.ARRAY
+    override fun isOrdered(): Boolean = false
```

Comment:
Why has this been removed?

Reply:
We don't have typing of the nodes, so this would throw an exception.